viewing_party/main.py

```diff
@@ -0,0 +1,135 @@
+# wave 1
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title == None or genre == None or rating == None:
+        new_movie = None
+    else:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched":[
+            movie
+        ]
+        }
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist":[
+            movie
+        ]
+        }
+    return user_data
+
+def watch_movie(user_data,title):
+    for movie in user_data['watchlist']:
+        if title == movie['title']:
+            user_data['watchlist'].remove(movie)
+            user_data['watched'].append(movie)
+            return user_data
+    return user_data
+
+# Wave 2
+
+def get_watched_avg_rating(user_data):
+    value = []
+    for movie in user_data["watched"]:
+        rating_num = movie['rating']
+        value.append(rating_num)
+    total = sum(value)
+    if total != 0:
+        average = total/len(value)
+    else:
+        average = 0.0
+    return average
+
+def get_most_watched_genre(user_data):
+    genre_list = []
+    if len(user_data['watched']) == 0:
+        popular_genre = None
+        return popular_genre
+    for movie in user_data['watched']:
+        genre = movie['genre']
+        genre_list.append(genre)
+        popular_genre = max(set(genre_list), key = genre_list.count)
```

Comment:
Hey Abigail, that is great! I'm glad you're utilizing more advanced methods and data structures. However, in the future, please add a comment that describes code that isn't covered in our curriculum.